packages/search/src/features/search/utils.ts

```diff
@@ -755,4 +755,16 @@ export const findPatientPrimaryIdentifier = (patient: Patient) =>
           'BIRTH_CONFIGURABLE_IDENTIFIER_3'
         ].includes(code)
     )
+  ) ??
+  // return registration numbers with a lower priority
+  findPatientIdentifier(
+    patient,
+    SUPPORTED_PATIENT_IDENTIFIER_CODES.filter(
+      (code) =>
+        ![
+          'BIRTH_CONFIGURABLE_IDENTIFIER_1',
+          'BIRTH_CONFIGURABLE_IDENTIFIER_2',
+          'BIRTH_CONFIGURABLE_IDENTIFIER_3'
+        ].includes(code)
+    )
```

Comment:
@Zangetsu101 should we invert the filter clauses from these both? At least I'm having a hard time parsing through the function.  So If I'm reading all of this right, it would be 

```ts
export const findPatientPrimaryIdentifier = (patient: Patient) =>
  findPatientIdentifier(
    patient,
    SUPPORTED_PATIENT_IDENTIFIER_CODES.filter((code) =>
      [
        'PASSPORT',
        'NATIONAL_ID',
        'MOSIP_PSUT_TOKEN_ID',
        'DECEASED_PATIENT_ENTRY',
        'BIRTH_PATIENT_ENTRY',
        'DRIVING_LICENSE',
        'REFUGEE_NUMBER',
        'ALIEN_NUMBER',
        'OTHER',
        'SOCIAL_SECURITY_NO'
      ].includes(code)
    )
  ) ??
  findPatientIdentifier(
    patient,
    SUPPORTED_PATIENT_IDENTIFIER_CODES.filter((code) =>
      [
        'BIRTH_REGISTRATION_NUMBER',
        'DEATH_REGISTRATION_NUMBER',
        'MARRIAGE_REGISTRATION_NUMBER'
      ].includes(code)
    )
  )
```

Reply:
Sure. That will probably be easier to follow